tests/integration/components/osf-footer/component-test.ts

```diff
@@ -0,0 +1,14 @@
+import { render } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | osf-footer', function(hooks) {
```

Comment:
If you change this to an arrow function, it'll pass without modifying the linting rules.